test/core/test/imports.test.ts

```diff
@@ -1,30 +1,30 @@
 import { expect, test } from 'vitest'
 
 test('dynamic relative import works', async() => {
-  const { timeout } = await import('./../src/timeout')
```

Comment:
Isn't it a named export I am not sure if I understand why we can do this - or are you checking the module reference to be the same?

Reply:
yes, this exactly what i am doing

Reply:
how about name the variables `module` so it could be less confusing? like

`literalImportedMod` and `stringImportedMod` - but you call it :)

Reply:
Done